components/CommunitySingle/CommunitySingle.js

```diff
@@ -97,7 +115,36 @@ function CommunitySingle(props = {}) {
             {`Find your ${props.data?.title}`}
           </Button>
         </Box>
-      </Hero>
+      </Styled.Hero>
+
+      {showNotifyMe && (
+        <Styled.NotifyMeSection>
+          <Box maxWidth={{ lg: '60%' }} mr={{ lg: 'l' }}>
+            <Box as="h3">Crew groups are closed for enrollment</Box>
+            <Box as="p" color="subdued">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin ut
+              imperdiet erat, sed sodales lorem.
+            </Box>
+          </Box>
+          <Box
+            display="flex"
+            flex={1}
+            flexDirection="row"
+            justifyContent="center"
```

Comment:
Could we try `justifyContent={{ _: 'center', lg: 'flex-end' }}`

Reply:
Updated in 5e40fa6